docs/api-reference/integration-guide.rst

```diff
@@ -45,7 +45,8 @@ Here are some tips.
 * JSON API: ``https://pypi.org/pypi/{name}/json`` returns the
   expected JSON response directly. See :doc:`json`.
 
-* XML-RPC API: see :ref:`changes-to-legacy-api`.
+* XML-RPC API: see :ref:`changes-to-legacy-api`. Will be deprecated in
+  the future; switch to the RSS or JSON APIs.
```

Comment:
Should we call to action to file issues for things that they use that only exists in the XMLRPC API?

Reply:
Good call! Fixed.